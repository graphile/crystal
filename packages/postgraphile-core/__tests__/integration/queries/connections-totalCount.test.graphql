query {
  a: allPeople {
    totalCount
  }
  b: allPeople {
    nodes {
      friends {
        totalCount
      }
    }
  }
  c: tableSetQuery {
    totalCount
  }
}
