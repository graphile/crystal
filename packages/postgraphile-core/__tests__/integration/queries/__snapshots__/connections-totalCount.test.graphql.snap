// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`execute 1`] = `
Object {
  "data": Object {
    "a": Object {
      "totalCount": 6,
    },
    "b": Object {
      "nodes": Array [
        Object {
          "friends": Object {
            "totalCount": 2,
          },
        },
        Object {
          "friends": Object {
            "totalCount": 2,
          },
        },
        Object {
          "friends": Object {
            "totalCount": 2,
          },
        },
        Object {
          "friends": Object {
            "totalCount": 2,
          },
        },
        Object {
          "friends": Object {
            "totalCount": 1,
          },
        },
        Object {
          "friends": Object {
            "totalCount": 0,
          },
        },
      ],
    },
    "c": Object {
      "totalCount": 6,
    },
  },
}
`;
